### PR TITLE
Adds simple search box, MessagingSearch component

### PR DESCRIPTION
**Pull request by webinista**
Work on department-of-veterans-affairs/messaging-fe#14
- Also adds Flexbox utility classes.

### CODE REVIEW

File: src/sass/base/_va.scss [hunk 1]
```diff
@@ -1981,3 +1981,23 @@ pre[class*='language-']  {
   border-left: 2px solid $color-primary;
   padding-left: 1rem;
 }
+
+// Flexbox related utilities 
+.va-flex {
+  display: -webkit-flex;
```
Comment (carlmjohnson) [inline src/sass/base/_va.scss]: Are we not using [autoprefixer](https://github.com/postcss/autoprefixer)?

Comment (webinista) [inline src/sass/base/_va.scss]: AFAIK, we are not.

Comment (carlmjohnson) [inline src/sass/base/_va.scss]: I don't really understand the webpack config format, but it doesn't look like it's in there. We should think about adding it later. I'll put in a ticket for it.

Comment (carlmjohnson) [inline src/sass/base/_va.scss]: #3235

**Comment by webinista** [comment]
Build is breaking because of a failing edu benefits test 😕

**Comment by akainic** [comment]
@webinista it's passing now!

**Comment by U-DON** [comment]
The search box should go below the folder name on desktop and below the compose button on mobile. But LGTM, since it's a WIP.

**Pull request merged**